opentelemetry-semantic-conventions/src/semconv/resource.rs

```diff
@@ -0,0 +1,882 @@
+// DO NOT EDIT, this is an auto-generated file
```

Comment:
Was this file added by mistake? seems to be a dup of src/resource.rs

Reply:
Left in by mistake, removing.